rest-server/src/main/java/io/github/nubesgen/configuration/NubesgenConfiguration.java

```diff
@@ -203,11 +205,23 @@ public boolean isRuntimeJava() {
             RuntimeType.SPRING.equals(this.getRuntimeType()) ||
             RuntimeType.SPRING_GRADLE.equals(this.getRuntimeType()) ||
             RuntimeType.QUARKUS.equals(this.getRuntimeType()) ||
+            RuntimeType.MICRONAUT.equals(this.getRuntimeType()) ||
+            RuntimeType.MICRONAUT_GRADLE.equals(this.getRuntimeType()) ||
             RuntimeType.JAVA.equals(this.getRuntimeType()) ||
             RuntimeType.JAVA_GRADLE.equals(this.getRuntimeType())
         );
     }
 
+    @JsonIgnore
+    public boolean isRuntimeMicronaut() {
```

Comment:
should not be better as a method of `RuntimeType`?

Reply:
This is used by the mustache templates, so I'm not sure it can be moved

Reply:
Oh yes this is purely for Mustache